hack/deploy.sh

```diff
@@ -5,13 +5,4 @@ set -xe
 source hack/config
 
 docker login -u="${DOCKER_USER}" -p="${DOCKER_PASS}"
-
-# This builds a multi-architecture image and uploads it, all in a
-# single step. The cached layers from the build step will be reused,
-# so ultimately this only assembles and pushes the manifest
-docker buildx build \
-    --progress=plain \
-    --push \
-    --platform="${DOCKER_PLATFORM}" \
-    --tag "${REGISTRY_TAG}" \
-    .
+docker push "${REGISTRY_TAG}"
```

Comment:
What do you recommend, duplicate the 'docker buildx' invocation in both scripts? Or are you suggesting the ideal workflow is users call deploy as well? That will take more work to sort out skipping the 'docker login' call when it's not required.

I left this bit unchanged in the latest push since I'm not sure what to do